src/settings/OrderNumber.js

```diff
@@ -40,26 +47,9 @@ class OrderNumber extends Component {
         getInitialValues={this.getInitialValues}
         label={label}
         moduleName={MODULE_ORDERS}
-      >
-        <Accordion
-          label={<FormattedMessage id="ui-orders.settings.poNumber" />}
-          separator={false}
-        >
-          <Row>
-            <Col xs={12}>
-              <Headline margin="none">
-                <FormattedMessage id="ui-orders.settings.poNumber" />
-              </Headline>
-              <Field
-                component={Checkbox}
-                label={<FormattedMessage id="ui-orders.settings.poNumber.editPONumber" />}
-                name="orderNumber"
-                type="checkbox"
-              />
-            </Col>
-          </Row>
-        </Accordion>
-      </this.configManager>
+        onBeforeSave={this.beforeSave}
+        configFormComponent={OrderNumberForm}
```

Comment:
Is there a reason to use `configFormComponent` ? What I see - it's just duplicates the code for form and save button. Previous way was less duplicating, IMHO.